infrastructure/cdn-in-a-box/traffic_ops/config.sh

```diff
@@ -98,7 +98,8 @@ cat <<-EOF >/opt/traffic_ops/app/conf/cdn.conf
         "max_db_connections": 20,
         "backend_max_connections": {
             "mojolicious": 4
-        }
+        },
+        "whitelisted_oauth_urls": ["*.xfinity.tv"]
```

Comment:
I don't think you meant for this to be here.

Reply:
oh shoot good catch. deleted it